### DISCUSSION TITLE
About the lowest electron energy that beamnrc can calculate.

### DISCUSSION THREAD

**Discussion by CQUswl** [nrc-cnrc/EGSnrc#1191] [Questions]
When I used beamnrc, I found that when the electron energy is below 10 kev, I can't get the result. Is it because the program limits the minimum energy? Is there a way to lift this restriction?

**Comment by rtownson** [comment]
In the transport parameters, you need to set ECUT and PCUT to adjust transport energy cutoffs for photons and electrons, respectively. The lowest possible is 1 keV, or `ECUT=0.512` and `PCUT=0.001`, since it is total energy in MeV. 

The materials also have secondary production thresholds built into the data. This is AE and AP. If you're using the 521icru.pegs4dat data, then it has AE,AP set to a 10keV cutoff. You'll need to create new data with AE and AP reduced, or switch to pegsless mode. The material.dat file has all of the same materials as 521icru.pegs4dat but prepared for pegless mode. Just add this to your input file, adjusting the path to the file. Note that UE and UP are upper thresholds, don't make them too huge.
```
:start media definition:
        ae = 0.512
        ue = 50.511
        ap = 0.001
        up = 50
 
 
        material data file=/yourpath/HEN_HOUSE/pegs4/data/material.dat
:stop media definition:
```
